gradle.properties

```diff
@@ -33,5 +32,7 @@ systemProp.http.proxyPort=0
 systemProp.http.proxyHost=
 android.useAndroidX=true
 android.enableJetifier=true
-android.enableR8=false
-android.jetifier.blacklist=bcprov-jdk15on
\ No newline at end of file
+android.jetifier.ignorelist=bcprov-jdk15on
```

Comment:
Changed **android.jetifier.blacklist** to **android.jetifier.ignorelist**.

Refer to this GitHub [comment](https://github.com/cashapp/paparazzi/issues/245#issue-975739503).